packages/core/README.md

```diff
@@ -234,6 +234,9 @@ MobileCore.setLogLevel(LogLevel.VERBOSE);
 
 - #### trackAction
 
+> [!IMPORTANT]  
+> trackAction is supported through [Edge Bridge](../edgebridge) and [Edge Network](../edge) extensions. 
+
```

Comment:
Should we call out Analytics Extension is not supported here?

Reply:
I think we should be good with covering that in the migration doc.